scripts/pretraining/data_preprocessing.py

```diff
@@ -8,8 +8,12 @@
 import multiprocessing
 
 from pretraining_utils import get_all_features
+from gluonnlp.base import get_repo_model_zoo_url
+from gluonnlp.utils.misc import download
 from gluonnlp.data.tokenizers import HuggingFaceWordPieceTokenizer
 
+VOCAB_PATH = 'google_electra_small/vocab-e6d2b21d.json'
```

Comment:
I suggest adding name of the backbone model so it will use the same tokenizer as the backbone.

Reply:
Could we have a  function called `get_tokenizer` that plays similar role as `get_backbone` but only return tokenizer and vocab according to the given backbone name

Reply:
Yes, we can.

Reply:
In fact, it may just be a thing wrapper around get_backbone because each backbone has a tokenizer.